src/scss/cs/button.scss

```
@@ -365,10 +365,8 @@ a:not(.btn):not(.nav-link) {
   }
 }
 
-strong {
-  a {
-    color: $dark-pink-300 !important;
-  }
+strong a, a strong {
+  color: $dark-pink-300 !important;
```

Since you've styled `.btn.btn-inline-link` essentially identically to a `strong a` now anyway, should we combine both of the css expressions in this file to the same line too? I think it's quite likely that any future changes would want to be applied to both now, since they're mostly the same thing.

```suggestion
strong a, a strong, .btn.btn-inline-link {
  color: $dark-pink-300 !important;
```

---

Yep, good idea. There's also the content links on the line above this. Not quite sure the link ones belong in `button.scss`, but imo consistency is more important and combining them as you suggest guarantees that.